### PR TITLE
feat: config set token

### PR DESCRIPTION
**Pull request by dimabru**
* Added config cmd package to deal with config commands
* Refactored `localConfig`
* Support command `datree config set [key] [value]` - changes key-value in config.yaml

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -12,3 +12,6 @@ create-bin:
 
 print-version:
 	go run -tags=staging -ldflags="-X github.com/datreeio/datree/cmd.CliVersion=0.0.1" main.go version
+
+set-token:
```
Comment (myishay) [inline Makefile:16]: Alt Text

File: cmd/config/set.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+import (
+	"fmt"
+
+	"github.com/datreeio/datree/bl/messager"
+	"github.com/spf13/cobra"
+)
+
+func NewSetCommand(ctx *ConfigCommandContext) *cobra.Command {
+	setCommand := &cobra.Command{
```
Comment (myishay) [inline cmd/config/set.go:11]: I think `config set` command should be restricted only for setting `token` for now

**Review by myishay — COMMENTED** [review]
looks cool! gave you 1 comment

File: cmd/config/set.go (hunk 2 above)
Comment (dimabru) [reply cmd/config/set.go:11]: this is intentional. config set will be able to change any key-value in config.yaml file

**Pull request merged**